micro-rdk/src/common/board.rs

```diff
@@ -85,6 +85,16 @@ pub trait Board: DoCommand {
         ))
     }
 
+    fn add_digital_interrupt_callback(
+        &mut self,
+        _pin: i32,
+        _intr_type: InterruptType,
+        _cb: Option<unsafe extern "C" fn(_: *mut core::ffi::c_void)>,
+        _arg: Option<*mut core::ffi::c_void>,
+    ) -> Result<(), BoardError> {
+        unimplemented!();
```

Comment:
shouldn't return unimplemented here. Either make it a mandatory method or return a safe default